WordPress/Classes/Utility/WPUserAgent.h

```diff
@@ -6,25 +6,16 @@
  */
 @interface WPUserAgent : NSObject
 
-#pragma mark - Changing the user agent
-
 /**
- *  @brief      Sets the App's user agent to be the default one.
+ *  @brief      Sets User-Agent header of all UIWebViews to be the WordPress one.
  */
-- (void)useDefaultUserAgent;
-
-/**
- *  @brief      Sets the App's user agent to be the WordPress one.
- */
-- (void)useWordPressUserAgent;
-
-#pragma mark - Getting the user agent
+- (void)useWordPressUserAgentInUIWebViews;
```

Comment:
This might not be the best name.  The UA will be used by all HTTP requests, not just by those made by a web view. Maybe `useWordPressUserAgent` instead?